spec/cypress/integration/admin-cip-stories.js

```diff
@@ -0,0 +1,94 @@
+describe("Admin user interaction with Core Induction Programme", () => {
+  beforeEach(() => {
+    cy.login("admin");
+  });
+
+  it("should show a download export button", () => {
+    cy.visit("/core-induction-programme");
+    cy.get("a.govuk-button").contains("Download export").should("exist");
+  });
+
+  it("should allow to edit year title", () => {
+    cy.appFactories([["create", "course_year"]]);
+
+    cy.visitYear();
+    cy.get("h1").should("contain", "Test Course year");
```

Comment:
In my opinion this (and all the assertions like it) would be better inside the `visit*` commands